tests/lib/snaps/account-control-consumer/bin/useradd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec /usr/sbin/useradd "$@"
```

Comment:
Does this work? I expected '--extrausers' here. It would be good to have tests like this for each command allowed by the interface. As mentioned, some of these commands may not be aware of '--extrausers' so please file bugs if they don't work correctly (and then disable the test with a TODO to reenable it when the bug is fixed).